src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/CaseLinksController.java

```diff
@@ -56,14 +58,17 @@ public ResponseEntity<CCDCallbackResponse> createCaseLinkAboutToSubmit(
             return ResponseEntity.status(FORBIDDEN.value()).build();
         }
         log.info("Adding case links");
+        ccdRequest.getCaseDetails().getCaseData().setHearingIsLinkedFlag(Boolean.TRUE.toString());
```

Comment:
I know the way we handle this in CCD is a little unconventional, but I'm thinking using "Yes" or "No" would be better here.. The other flags use this pattern and its more native to CCD fields

Reply:
change to yes